lib/draper/version.rb

```diff
@@ -1,3 +1,3 @@
 module Draper
-  VERSION = '4.0.2'
+  VERSION = '4.1.0-pre'
```

Comment:
It was planned to stay at v4.0. Other issues are already planned for v4.1.

Reply:
I just incremented it to 4.1 based on this comment: https://github.com/drapergem/draper/issues/921#issuecomment-2533606192. I haven't looked at the changes on the master branch too closely. Thoughts @y-yagi?

Reply:
I didn't notice the PR to be for the `master` branch, sorry. I was thinking of it still in the context of that bad cherry-picking.

I'm not sure what version describes our `master` branch best for now. For me, pre-release of v4.1 sounds good, while @y-yagi considers some commits to be of v5.0.